operator/gefyra/configuration.py

```diff
@@ -32,6 +32,11 @@ def __init__(self):
         self.STOWAWAY_STORAGE = config(
             "GEFYRA_STOWAWAY_STORAGE", cast=int, default=64
         )  # see https://github.com/gefyrahq/gefyra/issues/670
+        self.STOWAWAY_MAX_CONNECTION_AGE = config(
```

Comment:
Since the data type is 'string', this fails for improperly set values (for example, '6000s') in the config map. We should enforce _int_ throughout..